packages/web-components/src/components/cta-block/cta-block.scss

```diff
@@ -103,24 +107,37 @@
   display: grid;
   grid-template-columns: 1fr;
   grid-template-rows: auto;
+  row-gap: $carbon--spacing-07;
+  position: relative;
 
-  ::slotted(#{$dds-prefix}-cta-block-item) {
-    margin-top: $spacing-07;
-    border-bottom: 1px solid $gray-20;
-
+  // Hide last row's items' bottom border
+  &::after {
     @include carbon--breakpoint('md') {
-      margin-top: 0;
-      border: 0;
+      content: '';
+      background-color: $ui-background;
+      height: 3px;
+      position: absolute;
+      bottom: -1px;
+      left: 0;
+      right: 0;
+      pointer-events: none;
     }
   }
```

Comment:
just wondering, when is this pseudoelement visible? it seems like it's always hidden when it gets rendered to the DOM

![image](https://user-images.githubusercontent.com/8265238/145048005-b7e21815-553c-4445-a552-d8cc954f828c.png)

Reply:
There should only be borders between each row, and this pseudo element serves that goal by hiding the last row's items' bottom borders. There is a darker bottom border in the design specs, but my understanding is that comes from `.bx--cta-block__border`, which can be disabled by adding the `no-border` attribute to `<dds-cta-block>`. I just pushed a commit adding a knob to the CTA Block story so this can be seen.